.changeset/fresh-days-fold.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': major
+---
+
+Replaced the `List` component sizes **_mega_** and **_kilo_** with new values **_one_** and **_two_** respectively (🤖 _typography-sizes_).
```

Comment:
I would add this to the other, existing typography changeset.

Reply:
Makes more sense. Will do it. Thanks :)